torsiondrive/launch.py

```diff
@@ -64,24 +65,31 @@ def load_dihedralfile(dihedralfile, zero_based_numbering=False):
                 comment = line[1:].strip().lower()
                 if comment == 'zero_based_numbering':
                     zero_based_numbering = True
+                    if one_based_explicit is True:
+                        raise ValueError("Can not specify both zero_based_numbering and one_based_indexing.")
                 elif comment == 'one_based_numbering':
-                    if zero_based_numbering == True:
+                    one_based_explicit = True
```

Comment:
This flag checks the edge case if the user explicitly specified `"#one_based_numbering"` in the file then also  `"#zero_based_numbering"`. It now detects the conflict and raise an error.